Tools/CMake/Libraries.cmake

```diff
@@ -266,7 +287,8 @@ if(WIN32)
 
   if(EXISTS ${RTOOLS_LIBREADSTAT_H})
     message(CHECK_PASS "found")
-    message(STATUS "  ${RTOOLS_LIBREADSTAT_H}")
+    message(STATUS "Now copy ${RTOOLS_LIBREADSTAT_H} to source directory.")
+    configure_file("${RTOOLS_LIBREADSTAT_H}" "${CMAKE_SOURCE_DIR}/Desktop/data/importers/readstat/readstat.h" COPYONLY)
```

Comment:
Why did you add this?

Reply:
You will noticed I had deleted "[readstat.h](https://github.com/jasp-stats/jasp-desktop/pull/5841/files#diff-c0d8f99c10584ea56a77bd4b6488e326d5ec3e0bc1b2bf39ba18e305ae353f56)" from this PR.